tests/task.test.ts

```diff
@@ -368,6 +368,65 @@ describe.each([{ permission: 'Master' }, { permission: 'Admin' }])(
       expect(tasksUids[0]).toEqual(addDocumentsTask.taskUid)
     })
 
+    // filters error code: INVALID_TASK_TYPES_FILTER
+    test(`${permission} key: Try to filter on task types with wrong type`, async () => {
+      const client = await getClient(permission)
+
+      await expect(
+        // @ts-expect-error testing wrong argument type
+        client.getTasks({ types: ['wrong'] })
+      ).rejects.toHaveProperty(
+        'code',
+        ErrorStatusCode.INVALID_TASK_TYPES_FILTER
+      )
+    })
+
+    // filters error code: INVALID_TASK_STATUSES_FILTER
+    test(`${permission} key: Try to filter on statuses with wrong type`, async () => {
+      const client = await getClient(permission)
+
+      await expect(
+        // @ts-expect-error testing wrong argument type
```

Comment:
copy/paste I guess 👀

Reply:
Yes, it is an accurate comment 😅 Do you want me to change it?